salt/metalk8s/kubeadm/init/addons/coredns.sls

```diff
@@ -0,0 +1,123 @@
+{%- from "metalk8s/map.jinja" import coredns with context %}
+
+{% set kubeconfig = "/etc/kubernetes/admin.conf" %}
+{% set context = "kubernetes-admin@kubernetes" %}
+
+include:
+  - metalk8s.salt.minion.running
```

Comment:
We don't do this in the `kube-proxy` state. Not sure it's required: in the end, if the minion is not running, this state likely won't run either...

Reply:
this state will make sure salt-minion restarts, because there was a state that adds some config file to the minion, so true, no need for this anymore